cypress/integration/Tools/CreateModel.spec.js

```diff
@@ -6,8 +6,8 @@
 const models = require('../../support/Models')
 const homePage = require('../../support/components/HomePage')
 
-describe('Tools', () => {
-  it('Create Model', () => {
+describe('Create Model', () => {
+  it('Tools', () => {
```

Comment:
I believe Matt would like "it" changed to "test"

Reply:
What lead you to believe this?  

The function you use/choose corresponds how the tests are written.  If writing style is more BDD then I think `it` fits better if it's more traditional / unit then `test` might be more appropriate.